GoogleForCreators/web-stories-wp
# Right Click Menu: Shape menu

## Context

Right Click Menu. [Designs](https://www.figma.com/file/1YM3jA9h4QGc4xLdyaXIJS/Stories-Sprint-1.7?node-id=7347%3A60835)

## Summary

Add the right click menu for shapes.

## Relevant Technical Choices

n/a

## To-do

n/a

## User-facing changes

No new functionality. All functions in the menu already exist except for `Add Color to 'Saved Colors'`. That action is added in this pr: #8605.

## Testing Instructions

Go to a story and right click on a shape element. See that all the actions works as expected and look like these [designs](https://www.figma.com/file/1YM3jA9h4QGc4xLdyaXIJS/Stories-Sprint-1.7?node-id=7347%3A60835)

### QA






## Reviews

### Does this PR have a security-related impact?

no

### Does this PR change what data or activity we track or use?

no

### Does this PR have a legal-related impact?

no

## Checklist




---



Fixes #6137
